stencil-components/src/components/c4-button/c4-button.tsx

```diff
@@ -14,10 +13,10 @@ export class C4Button {
   @Prop() buttonClass?: 'secondary' | 'tertiary' | 'right' | 'left';
 
   private getClass(): string {
-    let className = "button";
+    let className = "c4-button";
 
     if (this.buttonClass) {
-      className += ` ${this.buttonClass}`;
+      className += ` c4-button--${this.buttonClass}`;
```

Comment:
[The same pattern as above](https://github.com/cloudfour/tpp-research-monorepo/pull/5/files#r572461644) can be applied here if you think it makes sense. 🙂

Reply:
Good call!